django/http/multipartparser.py

```diff
@@ -292,7 +294,7 @@ def handle_file_complete(self, old_field_name, counters):
         """
         for i, handler in enumerate(self._upload_handlers):
             file_obj = handler.file_complete(counters[i])
-            if file_obj:
+            if file_obj is not None:
```

Comment:
This is a separate issue, can we move it to a separate PR/commit? Also, test is required.

Reply:
Yes I think we need a separate ticket. I'm not sure if there is a valid use for custom handlers that return files with an empty filenames :thinking: I removed the 2nd commit.